doc/api/diagnostics_channel.md

```diff
@@ -227,6 +227,56 @@ diagnostics_channel.subscribe('my-channel', onMessage);
 diagnostics_channel.unsubscribe('my-channel', onMessage);
 ```
 
+#### `diagnostics_channel.tracingChannel(nameOrChannels)`
+
+<!-- YAML
+added:
+ - REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `nameOrChannels` {string|TracingChannel} Channel name or
+  object containing all the [TracingChannel Channels][]
+* Returns: {TracingChannel} Collection of channels to trace with
+
+Creates a [`TracingChannel`][] wrapper for the given
+[TracingChannel Channels][]. If a name is given, the corresponding tracing
+channels will be created in the form of `tracing:${name}:${eventType}` where
+`eventType` corresponds to the types of [TracingChannel Channels][].
+
+```mjs
+import diagnostics_channel from 'node:diagnostics_channel';
+
+const channelsByName = diagnostics_channel.tracingChannel('my-channel');
```

Comment:
I feel `channelsByName` is quite confusing. Wouldn't it be just `channels`?

Reply:
I needed to give it a unique name because the linter complains if both examples share the same variable name.